### PR TITLE
Add org.manaplus.ManaPlus

### PR DESCRIPTION
**Pull request by AsciiWolf**
The game seems to work fine. :-)

Upstream was [contacted](https://github.com/ManaPlus/ManaPlus/issues/33).

### CODE REVIEW

**Comment by AsciiWolf** [comment]
bot, build org.manaplus.ManaPlus

**Comment by AsciiWolf** [comment]
bot, build org.manaplus.ManaPlus

File: org.manaplus.ManaPlus.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
+app-id: org.manaplus.ManaPlus
+runtime: org.freedesktop.Platform
+runtime-version: '18.08'
+sdk: org.freedesktop.Sdk
+command: manaplus
+rename-desktop-file: manaplus.desktop
+rename-appdata-file: manaplus.appdata.xml
+rename-icon: manaplus
```
Comment (bilelmoussaoui) [inline org.manaplus.ManaPlus.yaml]: Always prefer to have a copy-icon: true just in case.

**Comment by bilelmoussaoui** [comment]
Any chances of getting those patches upstream?

File: org.manaplus.ManaPlus.yaml (hunk 1 above)
Comment (AsciiWolf) [reply org.manaplus.ManaPlus.yaml]: The game seems to work fine without the old icon file being kept. :-)

Comment (AsciiWolf) [reply org.manaplus.ManaPlus.yaml]: But thanks for the tip!

Comment (bilelmoussaoui) [reply org.manaplus.ManaPlus.yaml]: Perfect

**Comment by AsciiWolf** [comment]
@bilelmoussaoui Sure. I will send a pull requests for them to the upstream repo. :-)

**Comment by AsciiWolf** [comment]
bot, build org.manaplus.ManaPlus

**Comment by barthalion** [comment]
Upstream seemed vaguely interested? @4144, we would like to give you write access. I assume @AsciiWolf will help with maintenance, but general goal of Flathub is to be a place for application developers.

**Comment by 4144** [comment]
I will merge pr bit after

**Comment by barthalion** [comment]
bot, build org.manaplus.ManaPlus

**Comment by barthalion** [comment]
Repository has been created: https://github.com/flathub/org.manaplus.ManaPlus


Thanks!

**Comment by barthalion** [comment]
@4144, make sure to accept the invitation to get write access to the repo above.

**Pull request closed without merging**